src/test/groovy/graphql/execution/ExecutionParametersTest.java

```diff
@@ -0,0 +1,41 @@
+package graphql.execution;
+
+import graphql.language.Field;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Map;
+
+import static graphql.Scalars.*;
+import static graphql.execution.ExecutionParameters.newParameters;
+import static graphql.execution.TypeInfo.newTypeInfo;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
+import static org.junit.Assert.assertSame;
+
+public class ExecutionParametersTest {
```

Comment:
Ok ... then I misunderstood you, sorry. 
Yes, please change it: I don't want to mix different Test frameworks, a "not so perfect groovy syntax" is the lesser evil.